### PR TITLE
fix path when emitting file

### PR DESCRIPTION
**Pull request by jenius**
Weird edge case, but when there was a path from dumpDirs being added it had a leading slash where no other path did. This actually was messing up our static plugin in roots-mini.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -16,7 +16,8 @@ module.exports = function (content) {
   if (query.dumpDirs) {
     query.dumpDirs.forEach((d) => {
       const re = new RegExp(`^${d}`)
-      if (url.match(re)) { url = url.replace(d, '') }
+      // TODO: make sure this slash is still ok on windows
```
Comment (kylemac) [inline index.js]: Would running it through path.resolve be helpful to catch any windows irregularities?

Comment (jenius) [inline index.js]: Yeah that's probably the way to go. Let me add that now.

**Comment by kylemac** [comment]
👍

**Comment by kylemac** [comment]
@jenius i'll leave it to you to release the new version

**Comment by kylemac** [comment]
err, I guess we go off carrot/file-loader still

**Pull request merged**